include/OpenSpaceToolkit/Astrodynamics/Trajectory.hpp

```diff
@@ -268,7 +268,7 @@ class Trajectory
     static Array<State> computeStates(
         const std::function<physics::coordinate::Position(const Instant&)>& aPositionGenerator,
         const Array<Instant>& anInstantArray,
-        const Duration& aStepSize = Duration::Seconds(1e-6)
+        const Duration& aStepSize = Duration::Seconds(1e-2)
```

Comment:
Maybe we need to come up with a better mechanism to find this value because if its this sensitive then this value isn't going to work for orbits with shorter or longer periods or trajectories on the ground etc?